Can't compile teeworlds client on windows

One of the newer changes in the master does that, I am at windows 10 (with all features installed)
```
C:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um\winnt.h(2482): error C2338: Windows headers require the default packing option. Changing this can lead to memory corruption. This diagnostic can be disabled by building with WINDOWS_IGNORE_PACKING_MISMATCH defined.
NMAKE : fatal error U1077: "C:\PROGRA~2\MICROS~1\2019\COMMUN~1\VC\Tools\MSVC\1424~1.283\bin\Hostx86\x86\cl.exe": Rückgabe-Code "0x2"
Stop.
NMAKE : fatal error U1077: ""C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\bin\HostX86\x86\nmake.exe"": Rückgabe-Code "0x2"
Stop.
NMAKE : fatal error U1077: ""C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\bin\HostX86\x86\nmake.exe"": Rückgabe-Code "0x2"
Stop.
NMAKE : fatal error U1077: ""C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\bin\HostX86\x86\nmake.exe"": Rückgabe-Code "0x2"
Stop.
```

@Robyt3 thank you, I can compile again! This time I got some warnings, might be interesting:
```
C:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um\objidl.h(513): warning C4099: "IStorage": Geben Sie den zuerst unter Verwendung von "class" und jetzt unter Verwendung von "struct" gesehenen Namen ein
C:\Users\Marvin\Desktop\workspace\teeworlds\src\engine/shared/config.h(47): note: Siehe Deklaration von "IStorage"
C:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um\objidl.h(9984): warning C4099: "IStorage": Geben Sie den zuerst unter Verwendung von "class" und jetzt unter Verwendung von "struct" gesehenen Namen ein
C:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um\objidl.h(9984): note: Siehe Deklaration von "IStorage"
```
Sorry for german localization, the other error was in english, too 🤔

--- a/src/engine/client/input.cpp
+++ b/src/engine/client/input.cpp
@@ -25,14 +25,14 @@
 	#define SDL_JOYSTICK_AXIS_MAX 32767
 #endif
 
-// for platform specific features that aren't available or are broken in SDL
-#include "SDL_syswm.h"
-
 #if defined(CONF_FAMILY_WINDOWS)
 #define WIN32_LEAN_AND_MEAN
 #include <windows.h>
 #include <imm.h>
 #endif
+
+// for platform specific features that aren't available or are broken in SDL
+#include "SDL_syswm.h"
 
 void CInput::AddEvent(char *pText, int Key, int Flags)
 {

--- a/src/engine/client/input.cpp
+++ b/src/engine/client/input.cpp
@@ -29,6 +29,7 @@
 #include "SDL_syswm.h"
 
 #if defined(CONF_FAMILY_WINDOWS)
+#define WIN32_LEAN_AND_MEAN
 #include <windows.h>
 #include <imm.h>
 #endif

